patches/minecraft/net/minecraft/entity/passive/HorseArmorType.java.patch

```diff
@@ -0,0 +1,62 @@
+--- ../src-base/minecraft/net/minecraft/entity/passive/HorseArmorType.java
++++ ../src-work/minecraft/net/minecraft/entity/passive/HorseArmorType.java
+@@ -25,6 +25,14 @@
+         this.field_188587_f = "";
+     }
+ 
++    //Allows for textures located outside the vanilla horse armor folder
++    private HorseArmorType(String textureLocation, String hash, int armorStrengthIn)
```

Comment:
Please keep all new methods in a `/********* FORGE *********/` section at the bottom of the file.